src/extension/ln/webbtc/index.ts

```diff
@@ -6,15 +6,30 @@ type RequestInvoiceArgs = {
   defaultMemo?: string;
 };
 
+declare global {
```

Comment:
This is duplicated in 3 files and is going to be difficult to maintain, I think we should really consider extracting all the duplicated provider code